src/components/ErrorNotifications.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, Dispatch } from 'react';
+import cn from 'classnames';
+import { Errors } from '../types/Errors';
+
+type Props = {
+  errorMessage: Errors;
+  setErrorMessage: Dispatch<React.SetStateAction<Errors>>;
+  handleResetErrorMessage: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+  handleResetErrorMessage,
+}) => {
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => {
+        setErrorMessage(Errors.DEFAULT);
+      }, 3000);
```

Comment:
It's a good practice to clear timeouts in the cleanup function of useEffect to prevent memory leaks. This ensures that any pending timeouts are cleared if the component unmounts or if the dependencies change.